content/en/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping.md

```diff
@@ -207,25 +222,46 @@ When starting the kubelet, if the file specified by `--kubeconfig` does not exis
 ```
 {{< /note >}}
 
-Additionally, in 1.7 the kubelet implements __alpha__ features for enabling rotation of both its client and/or serving certs.
-These can be enabled through the respective `RotateKubeletClientCertificate` and `RotateKubeletServerCertificate` feature
-flags on the kubelet, but may change in backward incompatible ways in future releases.
+Additionally, in 1.7 the kubelet implements __beta__ features for enabling
+rotation of both its client and/or serving certs.  These can be enabled through
+the respective `RotateKubeletClientCertificate` and
+`RotateKubeletServerCertificate` feature flags on the kubelet and are enabled by
+default.
+
+`RotateKubeletClientCertificate` causes the kubelet to rotate its client
+certificates by creating new CSRs as its existing credentials expire. To enable
+this feature pass the following flag to the kubelet:
 
 ```
---feature-gates=RotateKubeletClientCertificate=true,RotateKubeletServerCertificate=true
+--rotate-certificates
 ```
 
-`RotateKubeletClientCertificate` causes the kubelet to rotate its client certificates by creating new CSRs as its existing
-credentials expire. `RotateKubeletServerCertificate` causes the kubelet to both request a serving certificate after
-bootstrapping its client credentials and rotate the certificate. The serving cert currently does not request DNS or IP
-SANs.
+`RotateKubeletServerCertificate` causes the kubelet to both request a serving
```

Comment:
Above you removed mentions of `selfnodeserver` which is needed to make this work. Either remove serving cert mentions everywhere or re-add them above.

Also, to make his work, user has to set [ServerTLSBootstrap](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/apis/config/types.go#L112) config field.

Reply:
`selfnodeserver` was removed. You need to write a custom approving controller for this to work, as stated in the note.